content/api/recipient-validation.apib

```diff
@@ -0,0 +1,35 @@
+FORMAT: 1A
+title: Recipient Validation
+description: Tool for validating email recipient addresses.
+label: Beta
+
+# Group Recipient Validation
+
+Recipient Validation is a tool for validating recipient email addresses. 
+An address will either be returned as valid or invalid. 
+Any invalid email address responses will also provide a reason why it is invalid.
+
+## Single Address Validation [GET /v1/recipient-validation/single/{address}]
+
+Validate a single email address.
+
++ Request
+
+    + Headers
+
+            Accept: application/json
+            Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
++ Parameters
+
+    + address (required, string, `valid@ddress.com`)
+
++ Response 200 (application/json)
```

Comment:
At least 1 invalid example should be added, since they include an additional field.